tests/mock_network/utils.rs

```diff
@@ -603,7 +603,7 @@ pub fn verify_section_invariants_for_node(node: &TestNode, elder_size: usize) {
 
     if !our_prefix.is_empty() {
```

Comment:
Relax the min elder size requirement in tests 

Looking at this, it seem that the problem is the test is broken. We should not get in situation where we would get an elder_size that is smaller. In this case, maybe the safe_section_size needs to increase then?

Reply:
I think the test failure was false positive. There was nothing wrong with the network, yet the test still failed. So I think relaxing the test requirements is the right thing to do, to make them closer to reality.

Reply:
The problem is the way we relax the test requirements, we are moving away from reality where we would never have a case of not having enough nodes to populate the elders (i.e if we lost all adults already we are already gone.

Reply:
I guess there are two ways to look at it then. My view is that the section is still functional so there is no reason for the test to fail. You are right that we don't expect a section to fall this low in the real world, but then that isn't something caused by the routing logic itself, rather by external circumstances. So maybe the actual right thing to do would be to modify the test to never cause this situation in the first place. Although that could be tricky.

That said, increasing `safe_section_size` would probably work too and would bring the test a bit closer to reality too. So I'm OK with doing that if you want me to.

Reply:
> That said, increasing safe_section_size would probably work too and would bring the test a bit closer to reality too. So I'm OK with doing that if you want me to.

Sounds good.